spec/controllers/companies_controller_spec.rb

```diff
@@ -41,14 +41,25 @@
     end
     it 'renders show page' do
       company = create(:company)
+      sign_in company
       get :show, params: { id: company.id }
       expect(response).to render_template("show.html.erb")
     end
+    it 'redirects a user trying to access another users show page' do
+      company1 = create(:company,
+      email: "test@gmail.com", id: 20)
+      company = create(:company,
+      email: "example@gmail.com", id: 100)
+      sign_in company
+      get :show, params: { id: 20 }
+      expect(:company).to redirect_to("/")
```

Comment:
no tsure why this is passing but it should be: `expect(response).to redirect_to("/")`